docs/README.md

```diff
@@ -3,3 +3,60 @@
 Caltrans Performance Measurement System
 
 PeMS is open-source software that is designed, developed, and maintained by [Compiler LLC](https://compiler.la) on behalf of Caltrans Traffic Operations.
+
+## Current Work
+We do sprint planning and track day-to-day work in our [Project Board](/https://github.com/orgs/compilerla/projects/3/views/1).
```

Comment:
We don't want the `/` in front of the URL on any markdown links, e.g. links should look like

```md
[Link text](https://example.com)
```

Reply:
On it, thank you!